<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChainRecognizer</class>
 <widget class="QWidget" name="ChainRecognizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="recognize">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Распознать цепочку</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>196</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите цепочку для распознавания</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chainsList">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>350</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>135</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Промежуточная таблица Т</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableT">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>89</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="chainOutput">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>89</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вывод цепочки</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
